src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -257,6 +234,11 @@ export const OwnerContainer = ({ asset }: { asset: GenieAsset }) => {
   )
 }
 
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  ${OpacityHoverState}
+`
+
```

Comment:
http://localhost:3000/#/nfts/asset/0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d/2974

Reply:
this was missing correct styles